### PR TITLE
freebsd, netbsd: add directory; pkg, pkgin: add page

### LINKED ISSUE #7077: page request: pkgin
Documentation :  [NetBsd.org](https://www.netbsd.org/docs/pkgsrc/introduction.html#intro.platforms)
> - "There is a lot of software freely available for Unix-based systems, which is usually available in form of the source code. Before such software can be used, it needs to be configured to the local system, compiled and installed, and this is exactly what The NetBSD Packages Collection (pkgsrc) does. pkgsrc also has some basic commands to handle binary packages, so that not every user has to build the packages for himself, which is a time-costly task."

 --
> [Related To This Page on Tldr-Flutter client](https://github.com/Techno-Disaster/tldr-flutter/issues/12)

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

---

This PR adds the `freebsd` and `netbsd` directories and pages for their package managers.

We definitely don't want to add a big amount of platform directories. I'd like to hear everyone's opinion on adding more BSDs. I think those are the three most popular ones (+OpenBSD), and I wouldn't add any more myself after this.
Each platform addition requires updating all the clients, so adding them all in one go would be ideal.

Discussed in #10698.
Closes #7077.

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Review by kbdharun — COMMENTED** [review]
The directory changes LGTM. Thanks.

**Nitpick**: I would suggest moving the install and remove examples to the top of the pages.

----

_Excerpts from the chatroom_ (OpenBSD addition discussion)

[Myself](https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$Ivo0XGUMbbe_9Zolqz_5nqg18qvT3fg_hDRUBxobsN4?via=gitter.im&via=matrix.org&via=one.ems.host)

> Someone seems to have opened a PR to add pages for openbsd in a new directory https://github.com/tldr-pages/tldr/pull/10698, which is suggested in https://github.com/tldr-pages/tldr/issues/816. What do you guys think, can we go ahead with the addition of new directory.


[By](https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$YKWe-UUpY0GZWrKoAuIdx7eyE8qFU_z6scyF2W4AElk?via=gitter.im&via=matrix.org&via=one.ems.host) @pixelcmtd

> Yea I wasn't sure about possibly adding so many new platforms (openbsd, freebsd, netbsd, ...), as having one bsd platform would be so nice, but also probably impossible, because, as some have already pointed out, the BSD-derivates are very different in some aspects (a good example of this actually is mktemp). So it just went on my mental list of things to reply to "later". I think adding an openbsd platform here (i.e. merging the PR, assuming the pages are good) is probably the right judgement

It is indeed better to have BSD platforms added in a single go than adding them later for updating the clients + These projects pass the notability criteria to be added to tldr.

File: pages/freebsd/pkg.md
[the remark this answers is not in the document]
Comment (acuteenvy) [reply pages/freebsd/pkg.md:1]: The subcommand for removing packages is `delete`.
```markdown
- Delete a package:

`pkg delete {{package}}`
```

**Comment by pixelcmtd** [comment]
> Each platform addition requires updating all the clients

I've read pretty much this exact take from multiple people in the discussions around BSD platforms and I'm not sure I understand what you're referring to. Shouldn't a well-written client be able to display any page from any platform, as long as it exists? I'm aware of the fact that some clients (I checked a few, apparently most clients) might choose to make platforms, for example, an enum type and thereby check it's a platform the client knows about. But I don't think that's the right solution, at least in most cases. In my opinion, clients should be able to look at the `pages` directory to find all platforms that actually exist. (and we could possibly suggest that in the client spec)

**Comment by kbdharun** [comment]
> I've read pretty much this exact take from multiple people in the discussions around BSD platforms and I'm not sure I understand what you're referring to. Shouldn't a well-written client be able to display any page from any platform, as long as it exists? I'm aware of the fact that some clients (I checked a few, apparently most clients) might choose to make platforms, for example, an enum type and thereby check it's a platform the client knows about. But I don't think that's the right solution, at least in most cases. In my opinion, clients should be able to look at the pages directory to find all platforms that actually exist. (and we could possibly suggest that in the client spec)

Agreed, this is something that we should define with a suggested clause in the client specification (will add it to the [tracker issue](https://github.com/tldr-pages/tldr/issues/11146)). However, it would probably be difficult to implement in a majority of the community clients as they are all different implementations in various programming languages. In the case of the official clients, a few ones like the web client from the in-browser app, Node client does require manually updating the platforms for various reasons like Unit tests (in Node client which we specify a template page), etc.

File: pages/netbsd/pkgin.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pkgin
+
+> Manage pkgsrc binary packages on NetBSD.
```
Comment (kbdharun) [inline pages/netbsd/pkgin.md:3]:
```suggestion
> Manage `pkgsrc` binary packages on NetBSD.
```

**Review by kbdharun — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**